snyk-operator-certified/helm-charts/snyk-monitor/values.yaml

```diff
@@ -83,6 +83,7 @@ limits:
 http_proxy:
 https_proxy:
 no_proxy:
+use_keepalive:
```

Comment:
I think we want this to be `true` otherwise I think keep alive will be disabled by default